app/resources/brave_generated_resources_en-GB.xtb

```diff
@@ -147,7 +147,7 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
 <translation id="4112587618563511470">Invalid sync code, please check and try again.</translation>
 <translation id="2339730291919608488">Leave Sync Chain</translation>
 <translation id="115699408012835975">If you reset Sync, you will have to re-enter a sync code from another device in order to sync with it. Are you sure you want to proceed?</translation>
-<translation id="339086854515247923">Are you sure you want to delete selected device from sync chain?</translation>
+<translation id="339086854515247923">Are you sure you want to remove the selected device from the sync chain?</translation>
```

Comment:
This file should not be changed, it is auto-generated and will be updated from the translator service. Please revert all the file.

Reply:
Reverted the file

Reply:
thanks @rahulsnkr